SONiC_Workshop_Agenda_2022.html

```diff
@@ -0,0 +1,268 @@
+<!DOCTYPE html>
+<html>
+<head>
+
+               <h1 style= "text-align:center;" "font-family:verdana;" > SONiC Workshop Agenda </h1>
+			   <h3 style="text-align:center;" "font-family:verdana;">Thursday, November 17</h3>
+
+
+<style> 
+
+  table, th, td {
+  border: 1px  black;
+  border-radius: 1px;
+  font-family:arial;
+  background-color:black;
+  text-align:center
+ }
+th, td {
+  background-color: white;
+}
+</style>
+</head>
+<body>
+
+<table>
+
+
+ 
+
+  <tr>
+    <th style="background-color:tomato;"> Start Time</th>
+	<th style="background-color:tomato;"> End Time</th>
+	<th style="background-color:tomato;"> Duration</th>
+	<th style="background-color:tomato;"> Topic</th>
+	<th style="background-color:tomato;"> Speakers</th>
+	<th style="background-color:tomato;"> Company</th>
+  </tr>
+  <tr>
+    <td>8:50 AM</td>
+    <td>9:00 AM</td>
+	<td>10</td>
+    <td>SONiC Workshop Opening</td>
```

Comment:
We had two workshops in 2022, Nov: we had "SONiC Workshop @ OCP Global Summit 2022", In December, we had "SONiC Workshop @ ONE" -- this document is for the latter. Let's clarify.

Reply:
Addressed